jenkins-builds/1.4.0-21.12-eiger

```diff
@@ -1,3 +1,4 @@
+ Ascent-0.8.0-cpeGNU-21.12.eb         --set-default-module
```

Comment:
```suggestion
 Ascent-0.8.0-cpeGNU-21.12.eb         --set-default-module
```

Reply:
Thanks @jfavre: I would suggest to insert `Ascent-0.8.0-cpeGNU-21.12.eb` together with the group of `cpeGNU` builds in the file `[1.4.0-21.12-eiger](https://github.com/eth-cscs/production/pull/2741/files#diff-e3b814c41bf45b157631cd710087c7b12e06dda89ee4460aa09180593db46b03)`